multilateralfund/settings.py

```diff
@@ -324,6 +325,10 @@
 SESSION_COOKIE_HTTPONLY = True
 CSRF_COOKIE_HTTPONLY = False
 
+CLAMD_ENABLED = env.bool("CLAMD_ENABLED", default=True)
```

Comment:
Consider adding the variables in .env.example too.

Reply:
Added to .env.example, also changed the default CLAMD_ENABLED to False, as I feel it should not be used by default outside production/staging environments.